application.py

```diff
@@ -392,8 +388,8 @@ def contest_drafts(contest_id):
     title = contest_info[0]["name"]
 
     return render_template("contest/draft_problems.html", title=title,
-                           data=db.execute("SELECT * FROM :cidinfo WHERE draft=1",
-                                           cidinfo=contest_id + "info"))
+                           data=db.execute("SELECT * FROM context_problems WHERE contest_id=:cid AND draft=1",
```

Comment:
misspelling: should be contest_problems, not context_problems

Reply:
I believe that's fixed in the latest commit. I'm still working on fixing bugs since a lot were generated because of the change.